### PR TITLE
improve torrent metadata display

### PR DESCRIPTION
**Pull request by ngoldman**
Now it's a little clearer which speed is upload and which is download, and you can also see total size and total downloaded in bytes so far.

screen shot 2016-03-05 at 10 43 26 am

### CODE REVIEW

**Comment by ngoldman** [comment]
This PR now also includes a fix to chromecast play (was pointing to wrong object in state) and a fix for an uncaught torrent-list rendering error.

File: renderer/index.js [hunk 1]
```diff
@@ -285,8 +285,8 @@ function deleteTorrent (torrent) {
 
 function openChromecast (torrent) {
   startServer(torrent, function () {
-    state.view.chromecast.play(state.server.networkURL, { title: 'WebTorrent — ' + torrent.name })
-    state.view.chromecast.on('error', function (err) {
+    state.view.devices.chromecast.play(state.server.networkURL, { title: 'WebTorrent — ' + torrent.name })
+    state.view.devices.chromecast.on('error', function (err) {
```
Comment (feross) [inline renderer/index.js]: Yeah, this looks like leftover from someone's merge conflicts :)

File: renderer/views/torrent-list.js [hunk 2]
```diff
@@ -49,16 +49,28 @@ function renderTorrent (state, dispatch, torrent) {
...
 function renderTorrentMetadata (torrent) {
+  var progressPercent = 0
+  var progressBytes = 0
+
+  if (torrent.progress) {
+    progressPercent = Math.floor(100 * torrent.progress)
+  }
+
+  if (torrent.length && torrent.progress) {
+    progressBytes = torrent.length * torrent.progress
```
Comment (feross) [inline renderer/views/torrent-list.js]: You can actually just use `torrent.downloaded` for this. See: https://webtorrent.io/docs

**Comment by feross** [comment]
LGTM! :rocket:

File: renderer/views/torrent-list.js (hunk 2 above)
Comment (ngoldman) [inline renderer/views/torrent-list.js]: Cool, thanks!

**Pull request merged**